org-brain.el

```diff
@@ -2223,7 +2239,9 @@ Helper function for `org-brain-visualize'."
                 'org-brain-sibling))
              (setq max-width (max max-width (current-column)))
              (newline (forward-line 1)))
-           (sort children-links org-brain-visualize-sort-function))
+           (if (member org-brain-no-sort-children-tag parent-tags)
```

Comment:
I don't really understand the use case for this. Why would you want a special sort for the children of an entry? Why specifically the children?

Reply:
Say I'm managing a project, with a bunch of sub-projects, and I have a child entry for each subproject; and say further that the sub-projects are sequential. In org-brain I want the sub projects to be sorted in file order relative to one another. This means that when the main project is visited, its children (i.e., the sub-projects) should appear in file order, and when one of the sub-projects is visited, its sibling list should also appear in file order. This is accomplished by tagging the project with `nosort`.

As far as I know, without this you have to come up with a naming scheme so that the sub-projects appear in order, which gets complicated if you want to add a new sub-project in the middle of the list.

Reply:
I see, it makes sense :)